### ISSUE TITLE
BS don't watch grandparent folder

### ISSUE THREAD

**Issue by Plou** [BrowserSync/grunt-browser-sync#118]
Hi,
I tried to watch to watch files up in the file's tree but it don't seems to work. I can only go up on level

- `../img/*` works fine
- `../../img/*` don't

Any ideas ?

**Comment by shakyShane** [comment]
Likely a bug/limitation with [chokidar](https://github.com/paulmillr/chokidar/issues) - please ask/open an issue with those awesome guys :)

**Comment by Plou** [comment]
I found where the problem came from ^^.
So it was a slight mislead from chokidar which  caused a wrong default configuration of browser-sync.

The regex to ignore dotfiles shared in the read me of paulmillr/chokidar match any occurence of `/.` which is found in `../../` (see paulmillr/chokidar#339).

I changed it in my local configuration but, it could be better to change the default value.

``` coffeescript
watchOptions:
  ignored: /^([.][^.\/\\])|([\/\\]+[.][^.])/
```
Is a pull request needed ?
